test/test_fakeredis.py

```diff
@@ -709,6 +709,12 @@ def test_setex_using_float(r):
         r.setex('foo', 1.2, 'bar')
 
 
+@pytest.mark.min_server('6.2')
+def test_setex_overflow(r):
```

Comment:
What is the difference between SETEX and SET_EX?

Reply:
setex corresponds to the redis SETEX command. set_ex corresponds to the redis SET command with the EX option.